grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -59,18 +72,23 @@ class ClientJobStatus {
         }
         new ClientJobStatus(
             jobExecution.id,
+            jobExecution.transactionID,
             jobExecution.startTime,
             jobExecution.endTime,
             jobExecution.exitStatus,
             jobExecution.jobParameters.getString(ClientBatchJob.PATH),
             timeTaken,
-            jobExecution.stepExecutions?.find { it.stepName == "clientJcrNodes" }?.writeCount ?: -1
+            (jobExecution.stepExecutions?.find { it.stepName == "clientJcrNodes" }?.writeCount ?: -1) as Integer
         )
     }
 
-
-    public static List<ClientJobStatus> getAll(@Nonnull JobExplorer explorer) {
-        if (explorer == null) throw new IllegalArgumentException("JobExplorer == null")
+    /**
+     * Get the status of all jobs ever run
```

Comment:
This says "all jobs ever run", but the comment inside the function says "that are currently running" -- which is it?